### PR TITLE
Get user visit count

### PR DESCRIPTION
**Pull request by tingyuchang**
## 這個 PR 的目的 / Purpose of this PR

Add API provides bbs site online user count

## 相關的 issue / Related Issues
https://github.com/Ptt-official-app/go-openbbsmiddleware/issues/263

### CODE REVIEW

File: cache/cache_utmp.go [hunk 1]
```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/Ptt-official-app/go-pttbbs/types"
 )
 
+func GetUTotal() (total int32) {
+	Shm.ReadAt(
```
Comment (chhsiao1981) [inline cache/cache_utmp.go:11]: can you change the function name as GetUTMPNumber?
There is UTotal in the SHM.

**Review by chhsiao1981 — CHANGES_REQUESTED** [review]
can you change the function name as GetUTMPNumber?

Thanks!

File: cache/cache_utmp.go (hunk 1 above)
Comment (chhsiao1981) [reply cache/cache_utmp.go:11]: https://github.com/Ptt-official-app/go-pttbbs/blob/main/cache/shm_raw.go#L28

It's actually "Number" in SHMRaw, but the comment says "user total". (copied from c-pttbbs)

Comment (tingyuchang) [reply cache/cache_utmp.go:11]: OK

**Review by chhsiao1981 — APPROVED** [review]
/lgtm

**Pull request merged**